m2py/utils/post_processing.py

```diff
@@ -361,6 +362,68 @@ def show_classification_correlation(labels, data, data_type, title_flag=True, sa
     pyplot.show()
 
 
+def get_gmm_background_label(gmm_labels, connected_labels):
```

Comment:
Gets the GMM background label. This label is the one with the fewest number of connected components.